akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ReplayingSnapshot.scala

```diff
@@ -71,7 +71,7 @@ private[akka] class ReplayingSnapshot[C, E, S](override val setup: BehaviorSetup
         .receiveSignal(returnPermitOnStop.orElse {
```

Comment:
I spotted a different issue here https://github.com/akka/akka/issues/28367
Let's do that a separate ticket.

Reply:
Sounds good.